src/newComponents/viewer/LeadPreview/Preview/index.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState, useEffect, useRef, useCallback } from 'react';
+import {
+    _cs,
+    isNotDefined,
+    isDefined,
+    isValidUrl,
+} from '@togglecorp/fujs';
+import {
+    AiOutlineShrink,
+    AiOutlineExpandAlt,
+} from 'react-icons/ai';
+import {
+    IoOpenOutline,
+} from 'react-icons/io5';
+import {
+    QuickActionButton,
+    QuickActionLink,
+    TextInput,
+    ImagePreview,
+} from '@the-deep/deep-ui';
+
+import {
+    MimeTypes,
+    isImageMimeType,
+    isDocMimeType,
+    isHTMLMimeType,
+    createUrlForGoogleViewer,
+} from './mimeTypes';
+
+import styles from './styles.scss';
+
+function isHttps(text?: string) {
+    if (!text) {
+        return false;
+    }
+    return !!(text || '').match(/^https:\/\//);
```

Comment:
```suggestion
    return text && !!text.match(/^https:\/\//);
```

You can even use .startsWith()